exercises/pangram/pangram_test.rb

```diff
@@ -4,61 +4,66 @@
 require 'minitest/autorun'
 require_relative 'pangram'
 
-# Test data version:
-# deb225e Implement canonical dataset for scrabble-score problem (#255)
-
+# Test data version: # 2adfe21
 class PangramTest < Minitest::Test
   def test_sentence_empty
-    str = ''
-    refute Pangram.is_pangram?(str)
+    # skip
+    phrase = ''
+    refute Pangram.pangram?(phrase), "#{phrase.inspect} is NOT a pangram"
```

Comment:
I will miss `is_pangram?` that introduces `alias` and `alias_method` and other solutions in discussions.